amundsen_application/static/js/ducks/feedback/types.ts

```diff
@@ -1,21 +1,21 @@
-/* SubmitFeedback */
 export enum SubmitFeedback {
-  ACTION = 'amundsen/feedback/SUBMIT_FEEDBACK',
+  REQUEST = 'amundsen/feedback/SUBMIT_FEEDBACK_REQUEST',
   SUCCESS = 'amundsen/feedback/SUBMIT_FEEDBACK_SUCCESS',
   FAILURE = 'amundsen/feedback/SUBMIT_FEEDBACK_FAILURE',
-}
+};
 export interface SubmitFeedbackRequest {
-  type: SubmitFeedback.ACTION;
-  data: FormData;
-}
+  type: SubmitFeedback.REQUEST;
+  payload: {
+    data: FormData;
+  };
+};
 export interface SubmitFeedbackResponse {
   type: SubmitFeedback.SUCCESS | SubmitFeedback.FAILURE;
-}
+};
 
-/* ResetFeedback */
 export enum ResetFeedback {
-  ACTION = 'amundsen/feedback/RESET_FEEDBACK',
-}
+  REQUEST = 'amundsen/feedback/RESET_FEEDBACK_REQUEST',
+};
 export interface ResetFeedbackRequest {
-  type: ResetFeedback.ACTION;
-}
+  type: ResetFeedback.REQUEST;
+};
```

Comment:
I'm not sure we need the `;` for defining interfaces. We should probably define an explicit convention for this.

Reply:
I've been using it because this is what I believe is supposed to be the result of our current tslint configurations, but we have an open issue to figure out why things aren't getting applied.

Reply:
I don't totally agree that all the semi-colons are necessary, but we should table the discussion for another time.

Reply:
Sounds good. In general my plan was always to rely on the most popular tslint configurations as opposed to personal preference for linting, because unless we make an objective choice we will have these kinds of discussions over and over as more folks get involved.

Until our lint problem is fixed all of my lint comments are pretty much nits and I don't plan on manually enforcing them on anybody. However once we have that fixed, my hope with that we can take a purely objective approach to code style where the discussions are based on whether or not a particular configuration is a hinderance or a benefit, and why.